### PR TITLE
VAGOV-6233: VAMC FE: Side nav always shows Pittsburgh, regardless of the VAMC, or if there's no VAMC

### PR DESCRIPTION
**Pull request by aurora-a-k-a-lightning**
## Description
appropriate menus are now mapped for va facilities

## Testing done
locally with dev and staging data

## Screenshots
altoona
image

altoona operating status
image

## Acceptance criteria
https://va-gov.atlassian.net/browse/VAGOV-6233

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by ethanteague — APPROVED** [review]

**Comment by jefflbrauer** [comment]
Good morning @department-of-veterans-affairs/frontend-review-group ! Could someone please have a look at this PR?

File: src/site/stages/build/drupal/page.js [hunk 1]
```diff
@@ -182,6 +182,53 @@ function getHubSidebar(navsArray, owner) {
   return { sidebar: {} };
 }
 
+// used to find the correct sidebar menu if the page belongs to a health care region
+function getFacilitySidebar(page, contentData) {
+  // for pages like New Releases, Stories, Events, and Detail Pages
+  const facilityPage =
+    page.fieldOffice &&
+    page.fieldOffice.entity &&
+    page.fieldOffice.entity.entityLabel;
```
Comment (U-DON) [inline src/site/stages/build/drupal/page.js]: This seems like a good place for optional chaining:

```
const facilityPage = page.fieldOffice.entity?.entityLabel;
```

**Review by U-DON — APPROVED** [review]

**Comment by billfienberg** [comment]
What is the Altoona route? I see that `/pittsburgh-health-care` works, but `/altoona-health-care` doesn't. 

no-altoona-health-care-route

**Comment by ahay-agile6** [comment]
@billfienberg altoona must be enabled in drupal and you have to use the - -pull-drupal flag when compiling. You also have to enable the altoona menu

**Comment by billfienberg** [comment]
> @billfienberg altoona must be enabled in drupal and you have to use the - -pull-drupal flag when compiling the front-end. You also have to enable the altoona menu. This gets updated everytime the CMS is deployed - altoona gets unpublished. So you have to republish it every deploy until it is published on production, which will be at least everyday because i think Elijah had nightly deploys for the CMS.

@ahay-agile6, thanks for the explanation. That explains why it wasn't working for me.

**Pull request merged**